### PR TITLE
DAOS-7018 Test: Container destroy with force option after test.

### PR DESCRIPTION
**Pull request by ravalsam**
IOR might be still have some open connection to the container, which failed
to destroy the container. So adding force option for container destroy.

Test-tag-hw-medium: pr der_enospace

Signed-off-by: Samir Raval <samir.raval@intel.com>

### CODE REVIEW

**Comment by johannlombardi** [comment]
Ready for review?

**Comment by ravalsam** [comment]
> Ready for review?

Not yet, there is one more issue I am waiting to be verified in Ci. Once that test is passed in Ci will add a reviewer.

**Comment by ravalsam** [comment]
enospace test passed in Ci https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-6714/16/artifact/Functional%20Hardware%20Medium/nvme/enospace.py/results.html

File: src/tests/ftest/nvme/enospace.py [hunk 1]
```diff
@@ -302,12 +306,13 @@ def test_enospace_time_with_fg(self):
         #Repeat the test in loop.
         for _loop in range(10):
             print("-------enospc_time_fg Loop--------- {}".format(_loop))
+            print(self.pool.pool_percentage_used())
```
Comment (saurabhtandan) [inline src/tests/ftest/nvme/enospace.py:309]: Minor: It's preferred to use self.log.info() instead of print()

**Review by saurabhtandan — APPROVED** [review]
LGTM

File: src/tests/ftest/nvme/enospace.py (hunk 1 above)
Comment (ravalsam) [reply src/tests/ftest/nvme/enospace.py:309]: I will update in the next PR.

**Review by kccain — APPROVED** [review]

**Review by rpadma2 — APPROVED** [review]

**Pull request merged**